build_rust.py

```diff
@@ -0,0 +1,19 @@
+# A build script for poetry that adds the rust extension.
+
+import os
+
+from setuptools_rust import Binding, RustExtension
+
+
+def build(setup_kwargs):
```

Comment:
Is mypy going to check this file? Type hints would be nice, but I won't block this on having them.

Reply:
Alternatively stick it in `scripts-dev` and it'll be covered with everything else :angel:

Reply:
I don't think it can go in scripts-dev as it needs to be in the source to build the package?